### ISSUE TITLE
Observing '404' Error for profile-collection APIs with HUAWEI RTN 950

### ISSUE THREAD

**Issue by Ziabatcha** [openBackhaul/MicroWaveDeviceInventory#872]
MWDI: Tag v1.1.3

Observing ERROR '404' for HUAWEI RTN 950 model with APIs

 /core-model-1-4:network-control-domain=cache/control-construct={mountName}/profile-collection

/core-model-1-4:network-control-domain=cache/control-construct={mountName}/profile-collection/profile={uuid}

Note: we are able to see the response of these APIs fromLIVE but not from CACHE

{
  "code": 404,
  "message": "Field not found: profile-collection"
}

```
curl -X GET "http://xxx:xx/core-model-1-4:network-control-domain=cache/control-construct=513250004/profile-collection" -H "accept: application/json" -H "user: User Name" -H "originator: Resolver" -H "x-correlator: 550e8400-e29b-11d4-a716-446655440000" -H "trace-indicator: 1.3.1" -H "customer-journey: Unknown value" -H "operation-key: Operation key not yet provided."

http://xx:xxx/core-model-1-4:network-control-domain=cache/control-construct=513250004/profile-collection



```

```

curl -X GET "http://xxx:xx/core-model-1-4:network-control-domain=cache/control-construct=513250004/profile-collection/profile=PROFILE-WRED-2-service-green" -H "accept: application/json" -H "Authorization: Basic YWRtaW46YWRtaW5AMTIz"


http://xxx:xx/core-model-1-4:network-control-domain=cache/control-construct=513250004/profile-collection/profile=PROFILE-WRED-2-service-green

```

**Comment by nardecchia-siae** [comment]
Just verified in .21 VM with the same version test_v1.1.3 without any problem.

**Request:**
```
curl -X GET "http://172.28.127.21:3008/core-model-1-4:network-control-domain=cache/control-construct=513250004/profile-collection" -H "accept: application/json" -H "user: User Name" -H "originator: Resolver" -H "x-correlator: 550e8400-e29b-11d4-a716-446655440000" -H "trace-indicator: 1.3.1" -H "customer-journey: Unknown value" -H "operation-key: n.a."
```
**Response:**
```
{
  "core-model-1-4:profile-collection": {
    "profile": [
      {
        "uuid": "PROFILE-WRED-2-service-green",
        "wred-profile-1-0:wred-profile-pac": {
          "wred-profile-configuration": {
            "threshold-low": 32,
            "gentle-wred-is-on": false,
            "drop-probability-at-threshold-low": 40,
            "drop-probability-at-threshold-high": 100,
            "profile-name": "service wred",
            "coloring-is-on": true,
            "threshold-gentle": -1,
            "sensitivity": -1,
            "threshold-high": 80
          },
          "wred-profile-capability": {
            "sensitivity-setting-is-avail": false,
            "coloring-is-avail": true,
            "available-buffer-size": 1048320,
            "gentle-wred-is-avail": false,
            "drop-probability-at-threshold-low-is-avail": true
          }
        },
        "lifecycle-state": "core-model-1-4:LIFECYCLE_STATE_INSTALLED",
        "operational-state": "core-model-1-4:OPERATIONAL_STATE_ENABLED",
        "administrative-state": "core-model-1-4:ADMINISTRATIVE_STATE_UNLOCKED",
        "administrative-control": "core-model-1-4:ADMINISTRATIVE_CONTROL_UNLOCK",
        "profile-name": "wred-profile-1-0:PROFILE_NAME_TYPE_WRED_PROFILE"
      },
      {
        "uuid": "PROFILE-WRED-2-port-green",
        "wred-profile-1-0:wred-profile-pac": {
          "wred-profile-configuration": {
            "threshold-low": 480,.........
```

**Request:**
```
curl -X GET "http://172.28.127.21:3008/core-model-1-4:network-control-domain=cache/control-construct=513250004/profile-collection/profile=PROFILE-WRED-2-service-green" -H "accept: application/json" -H "user: User Name" -H "originator: Resolver" -H "x-correlator: 550e8400-e29b-11d4-a716-446655440000" -H "trace-indicator: 1.3.1" -H "customer-journey: Unknown value" -H "operation-key: n.a."
```
**Response:**
```
{
  "core-model-1-4:profile": [
    {
      "uuid": "PROFILE-WRED-2-service-green",
      "wred-profile-1-0:wred-profile-pac": {
        "wred-profile-configuration": {
          "threshold-low": 32,
          "gentle-wred-is-on": false,
          "drop-probability-at-threshold-low": 40,
          "drop-probability-at-threshold-high": 100,
          "profile-name": "service wred",
          "coloring-is-on": true,
          "threshold-gentle": -1,
          "sensitivity": -1,
          "threshold-high": 80
        },
        "wred-profile-capability": {
          "sensitivity-setting-is-avail": false,
          "coloring-is-avail": true,
          "available-buffer-size": 1048320,
          "gentle-wred-is-avail": false,
          "drop-probability-at-threshold-low-is-avail": true
        }
      },
      "lifecycle-state": "core-model-1-4:LIFECYCLE_STATE_INSTALLED",
      "operational-state": "core-model-1-4:OPERATIONAL_STATE_ENABLED",
      "administrative-state": "core-model-1-4:ADMINISTRATIVE_STATE_UNLOCKED",
      "administrative-control": "core-model-1-4:ADMINISTRATIVE_CONTROL_UNLOCK",
      "profile-name": "wred-profile-1-0:PROFILE_NAME_TYPE_WRED_PROFILE"
    }
  ]
}
```

**Comment by Ziabatcha** [comment]
Issue not observed with latest package test_v1.1.5.
Hence closing the issue.
[Uploading new 1.json…]()
